test/james.testing.clss.pas

```diff
@@ -44,6 +44,8 @@ interface
   {$ELSE}
     TTestCase = TestFramework.TTestCase;
   {$ENDIF}
+  ITest = James.Testing.ITest;
+  ITestSuite = James.Testing.ITestSuite;
```

Comment:
I believe it's simpler change the uses lile bellow.
We're creating a wraper for both frameworks (FPCUnit and DUnit) so, no user should use units from these frameworks directly, ie, there is no conflict.
```
uses
  {$IFDEF FPC}
    fpcunit,
    James.Testing.FPC,
  {$ELSE}
    TestFramework,
    James.Testing.Delphi,
  {$ENDIF}
  James.Testing;
```

What do you think?